code/game/gamemodes/miniantags/abduction/abductee_objectives.dm

```diff
@@ -148,3 +148,68 @@
 
 /datum/objective/abductee/sixthsense
 	explanation_text = "You died back there and went to heaven... or is it hell? No one here seems to know they're dead. Convince them, and maybe you can escape this limbo."
+
+/datum/objective/abductee/forbiddennumber
+	explanation_text = "Ignore anything in a set number of"
+
+/datum/objective/abductee/forbiddennumber/New()
+	..()
+	var/number = pick(list("two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"))
+	explanation_text +=" [number], they don't exist."
+
+/datum/objective/abductee/buddy
+	explanation_text = "Being alone and in large groups are both frightening.  Try to be alone with only one other person whenever possible."
+
+/datum/objective/abductee/finality
+	explanation_text = "Death should be final and modern medicine disrupts the natural order. Don't allow anyone to be revived."
+
+/datum/objective/abductee/mispronounced
+	explanation_text = "No matter how they say it, other people keep mispronouncing your name. Be sure to correct them whenever possible."
+
+/datum/objective/abductee/bald
+	explanation_text = "There are alien parasites masquerading as people's hair.  Save people from this invasion."
+
+/datum/objective/abductee/one
+	explanation_text = "There is only one other person in existence, he is just really good at pretending to be multiple people."
+
+/datum/objective/abductee/outlaw
+	explanation_text = "You have infiltrated this station as a space renowned space law, commit as many minor crimes as possible while remaining unnoticed by security."
```

Comment:
Text needs some updating, but this seems much better than it was before

Reply:
Fixed the space law/outlaw typo